backend/accounts/helper.py

```diff
@@ -1,104 +1,129 @@
 from django.conf import settings
 from django.template.loader import get_template
-from django.core.mail import EmailMultiAlternatives, send_mail
-from django.utils.http import int_to_base36
-from rest_framework import viewsets, status, filters
+from django.core.mail import send_mail
 from django.db.models import Q, Prefetch
-from django.db.models.signals import pre_save, post_save
+from django.db.models.signals import post_save
 from django.dispatch import receiver
 from django.contrib.contenttypes.models import ContentType
-from decimal import Decimal
 from django.contrib.auth.tokens import PasswordResetTokenGenerator
 from builtins import hasattr
-from postmarker.core import PostmarkClient
+
+# from postmarker.core import PostmarkClient
 
 from django.contrib.auth.models import User
 from plats.models import Plat, Lot, PlatZone
-from .models import Account, Agreement, AccountLedger, Payment, Project, ProjectCostEstimate, Profile
+from .models import (
+    Account,
+    Agreement,
+    AccountLedger,
+    Payment,
+    Profile,
+)
 from plats.utils import calculate_lot_balance, calculate_plat_balance
 
-postmark = PostmarkClient(server_token=settings.POSTMARK_API_KEY)
+# postmark = PostmarkClient(server_token=settings.POSTMARK_API_KEY)
+
 
 def send_password_reset_email(user, token):
-    text_template = get_template('emails/password_reset.txt')
-    html_template = get_template('emails/password_reset.html')
+    text_template = get_template("emails/password_reset.txt")
+    html_template = get_template("emails/password_reset.html")
 
     context = {
-        'user': user,
-        'baseURL': settings.BASE_URL,
-        'uid': user.id,
-        'token': token,
+        "user": user,
+        "baseURL": settings.SITE_DOMAIN,
+        "uid": user.id,
+        "token": token,
     }
 
     text_content = text_template.render(context)
     html_content = html_template.render(context)
 
-    subject, from_email = 'LFUCG Exactions: Password Reset', settings.DEFAULT_FROM_EMAIL
+    subject, from_email = "LFUCG Exactions: Password Reset", settings.DEFAULT_FROM_EMAIL
 
-    postmark.emails.send(
-        From=settings.DEFAULT_FROM_EMAIL,
-        To=[user.email],
-        Subject='LFUCG Exactions: Password Reset',
-        HtmlBody=html_content,
-    )
+    # postmark.emails.send(
```

Comment:
I think it's safe to remove all these dead versions of email ending once they've been tested / verified. Good call on switching to using the email backend that's configured.

Reply:
done